processor/src/main/java/com/workday/autoparse/json/codegen/JsonObjectParserTableGenerator.java

```diff
@@ -121,9 +126,7 @@ private void writeMapField(JavaWriter writer) throws IOException {
         for (Map.Entry<String, TypeElement> entry :
                 discrimValueToClassRequiringGeneratedParserMap.entrySet()) {
             String discriminationValue = entry.getKey();
-            String parserQualifiedName = MetaTypeNames.constructTypeName(entry.getValue(),
-                                                                         GeneratedClassNames
-                                                                                 .PARSER_SUFFIX);
+            String parserQualifiedName = MetaTypeNames.constructTypeName(entry.getValue(), GeneratedClassNames.PARSER_SUFFIX);
```

Comment:
do you need the parser_suffix? 
i had to get rid of it for the xml version.

Reply:
I do need it, otherwise I get an error on generate parse maps.